code/iaas/logic/src/main/java/io/cattle/platform/process/containerevent/ContainerEventCreate.java

```diff
@@ -40,12 +46,17 @@
 
 import org.apache.commons.lang3.BooleanUtils;
 import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import com.netflix.config.DynamicBooleanProperty;
 
 @Named
 public class ContainerEventCreate extends AbstractDefaultProcessHandler {
 
+    public static final String AGENT_ID = "agentId";
+    private static final String INSTANCE_INSPECT_EVENT_NAME = "compute.instance.inspect";
+    private static final String INSTANCE_INSPECT_DATA_NAME = "instanceInspect";
     private static final DynamicBooleanProperty MANAGE_NONRANCHER_CONTAINERS = ArchaiusUtil.getBoolean("manage.nonrancher.containers");
```

Comment:
I got input from Darren once to try not to use "rancher" name in our internal naming; but use cattle instead..as rancher is a product name, and can get changed